cockatrice/src/chatview.cpp

```diff
@@ -195,6 +195,10 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
                 break;
             // you have been mentioned
             if (message.toLower().startsWith(mention)) {
+                QColor customColor;
+                customColor.setNamedColor("#" + settingsCache->getChatMentionColor());
+                mentionFormat.setBackground(customColor.isValid() ? QBrush(customColor) : QBrush(DEFAULT_MENTION_COLOR));
```

Comment:
I think I'd like to see a `QColor getMentionColor()` method since you repeat this code